Fixed README grammar.

Knucklehead-McSpazatron opened the pull request:
Fixed the README's capitalization, punctuation, etc.

pokeheadroom commented:
Wow! This would probably have to be one of the most random pull requests i've ever gotten, however i'm glad you took the time to fix up my grammar... since well looking at just how long has it been since my last edit to the file i doubt i was ever going to get to do this myself. 

Much appreciated.

Knucklehead-McSpazatron commented:
> Wow! This would probably have to be one of the most random pull requests i've ever gotten, however i'm glad you took the time to fix up my grammar... since well looking at just how long has it been since my last edit to the file i doubt i was ever going to get to do this myself.
> 
> Much appreciated.

Thanks, I had been following this repo for a while and the grammar bothered me, no offense, so I opened a PR and fixed it. I also fixed some links so they appear on the names of the contributors instead of next to them.
